portfolio/src/main/webapp/index.html

```diff
@@ -68,9 +73,19 @@ <h6>Some Challenges/Stories About my Projects</h6>
       </ul>
       <h5>Fun Images in My Gallery</h5>
       <p>Click<a href="Gallery.html"> here</a> to take a look at my gallery.</p>
-      <p>Click here to see some of my best quotes:</p>
-      <button onclick="getQuotesUsingAsyncAwait()">let's do this</button>
-      <div id="quote-container"></div>
+      
+      <form action="/data" method="POST">
+        <label for="bquote">Name</label><br>
+        <input type="text" id="pname" name="pname"><br>
```

Comment:
Why pname? What means the p? Avoid abbreviations in code, that allows a better readability for other coders and also for you in the future.